service/templates/admin.html

```diff
@@ -133,6 +133,17 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 											</tr>
 										</thead>
 										<tbody>
+											<tr>
```

Comment:
Can we move this lower down in the list? The top list of flags are our "things are bad and we need to shut them off quick" kinds of actions.